certified-connectors/PowerBI Robots/apiDefinition.swagger.json

```diff
@@ -0,0 +1,184 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "PowerBI Robots",
+        "description": "PowerBI Robots connector to execute the internal API across the multiple Power Platform products",
+        "version": "1.0",
+        "contact": {
+            "name": "DevScope Support",
+            "url": "https://devscope.net/contacts/",
+            "email": "support@devscope.net"
+        }
+    },
+    "host": "powerbitiles.com",
+    "basePath": "/PBIRobots",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [],
+    "produces": [],
+    "paths": {
+        "/api/v1/playlist.enable": {
+            "post": {
+                "responses": {
+                    "default": {
+                        "description": "default",
+                        "schema": {},
```

Comment:
@AntonioArantesDevscope Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid response schema definition for all the operations in the swagger.

Reply:
commited on Identation, response and missing properties corrections